compat/src/index.js

```diff
@@ -24,28 +19,17 @@ import {
 	useContext,
 	useDebugValue
 } from 'preact/hooks';
+import { PureComponent } from './PureComponent';
+import { memo } from './memo';
+import { forwardRef } from './forwardRef';
+import { Children } from './Children';
 import { Suspense, lazy } from './suspense';
-import { assign, removeNode } from '../../src/util';
 import { SuspenseList } from './suspense-list';
+import { createPortal } from './portals';
+import { render, REACT_ELEMENT_TYPE } from './render';
```

Comment:
Fiddling with the order of the array imports led to some byte savings gains here